src/rpc/protocol.cpp

```diff
@@ -135,3 +135,20 @@ void DeleteAuthCookie()
     }
 }
 
+std::vector<UniValue> JSONRPCProcessBatchReply(const UniValue &in, size_t num)
+{
+    if (!in.isArray()) {
+        throw std::runtime_error("Batch must be an array");
+    }
+    std::vector<UniValue> batch(num);
+    for (size_t i=0; i<in.size(); ++i) {
+        const UniValue &rec = in[i];
+        if (!rec.isObject())
```

Comment:
please also add braces to these if blocks

Reply:
Done